src/components/Card/Card.tsx

```diff
@@ -1,15 +1,20 @@
 import React, { Component } from 'react';
 import './Card.css';
 
-
-export default class Card extends Component {
-    constructor(props) {
+export interface CardProps {
+    info: any,
+    buttonDisplay: any,
```

Comment:
redefine type for buttonDisplay, which is a prop

Reply:
ya